src/components/authoring.tsx

```diff
@@ -65,9 +65,16 @@ export const defaultAuthoring: ConnectedBioAuthoring = {
   },
   breeding: {
     instructions: "",
-    breedingType: "litter"
+    breedingType: "litter",
+    nestPairs: [
```

Comment:
Because of the fact that we're going to be randomizing the sex of the mice, I don't think that this needs to be a part of the authoring. I think the details of how to generate the pairs could be hard-coded in the `breeding.ts` model itself, perhaps in `createBreedingModel` (like `createOrganismsModel`).

(That is, unless you've understood that the authors might ever want to change the pairs, but so far I haven't heard this.)

Reply:
I moved the nestPair creation to createBreedingModel.